source/_components/media_player.bluesound.markdown

```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+title: "Bluesound"
+description: "Instructions how to integrate Bluesound devices into Home Assistant."
+date: 2017-04-21 19:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: bluesound.png
+ha_category: Media Player
+featured: true
+ha_release: 0.44
+ha_iot_class: "Local Polling"
+---
+
+The `bluesound` platform allows you to control your [Bluesound](http://www.bluesound.com/) HiFi wireless speakers and audio components from Home Assistant.
+
+If you want to automatically discover new devices, just make sure you have discovery: in your configuration.yaml file. To manually add a Bluesound device to your installation, add the following to your configuration.yaml file:
+
+
+```yaml
+# Example configuration.yaml entry with manually specified addresses
+media_player:
```

Comment:
Generalize the examples to have the minimal amount of configuration needed.
Feel free to add a advanced example if you feel it's necessary below.